### ISSUE TITLE
🐛 BUG: In local dev mode, making more than 50 subrequests results in an error even though worker runs on Unbound model

### ISSUE THREAD

**Issue by s1mple** [cloudflare/wrangler2#2084]
### What version of `Wrangler` are you using?

2.1.10

### What operating system are you using?

Mac

### Describe the Bug

In local dev mode, making more than 50 subrequests in a worker returns an html page with an error, even though the service being developed is setup to run under the Unbound usage model in production (with 1000 max subrequests per request as described in these docs: https://developers.cloudflare.com/workers/platform/limits/#how-many-subrequests-can-i-make). 

The specific error text is as follows:
```
Error
Too many subrequests. Workers can make up to 50 subrequests per request. 
A subrequest is a call to fetch(), a redirect, or a call to any Cache API method.
```

<b>Proposed solution:</b> Could there be a flag we pass to wrangler dev, such as `--unbound`, to simulate the unbound usage model limits when we are developing locally?

**Comment by JacobMGEvans** [comment]
Have you tried running in `--local` mode? The preview edge used in `remote` mode is limited in some behaviors. We can discuss internally if something like the `--unbound` flag could be possible.

**Comment by s1mple** [comment]
Yes, sorry, I was running this in --local mode when the error occurred. This issue's not a biggie, just figured I'd document it here for reference

**Comment by penalosa** [comment]
This should be fixed in `wrangler@3`, which uses the open source `workerd` runtime in local mode
